lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -511,11 +511,41 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     }
     // TODO: Take approximation into account.
     std::string approximate;
-    if (!matchPattern(gelu.getApproximate(), m_TorchConstantStr(approximate)) ||
-        approximate != "none")
+    if (!matchPattern(gelu.getApproximate(), m_TorchConstantStr(approximate))) {
```

Comment:
Nit: Given there is only one shared operator (`MulFOp`). We might as well check cases and bail early. It makes the control flow a little easier to read.

```
if (approximate == "none") {
  Value multiplier = buildUnitNormalCdf(b, loc, payloadArgs[0]);
  return b.create<arith::MulFOp>(loc, payloadArgs[0], multiplier);
}

if (approximate == "tanh") {
  ....
  return b.create<arith::MulFOp>(loc, payloadArgs[0], multiplier);
}


gelu.emitError("unimplemented: approximate value should be none or tanh");
return nullptr;
```